app/src/ui/lib/list/list.tsx

```diff
@@ -1065,6 +1066,14 @@ export class List extends React.Component<IListProps, IListState> {
             event,
           })
         }
+      } else if (
```

Comment:
@sergiou87 You may be rightly weary of changes to this generalized `List` component that is the basis for most if not all lists in the app.  Just for some piece of mind I am adding this screen shot to show that selectionMode of range is only used for the commit list.  Thus, this 'should' not impact other lists... (tho out of caution I did regression test the changes list as it make significant use of the methods I modified here) 

![image](https://user-images.githubusercontent.com/75402236/110117239-97c08b80-7d86-11eb-806a-74ecb442c3f2.png)

(Also in case it is not readily clear.. tho I hope my comments indicate it. The reason for this change is that if a user had multiple commits selected and then pressed down on one to start dragging the multiple commits, it would invoke a single selection again. Tho the state at the time of the drag still had the multiple commits so it "worked" it appeared like they were only dragging one)